### PR TITLE
fauna_ground_counts_v2

### PR DESCRIPTION
**Pull request by arunherb**
Updated the Fauna Ground Counts Vocabs Mapping based on version - 2 changes.

### CODE REVIEW

File: vocab_files/attribute_concepts/outlook-zone-left-direction.ttl [hunk 1]
```diff
@@ -9,9 +9,9 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
...
     dcterms:source "McCallum K, Bignall J, Cox B, O’Neill S, Sparrow B. (2023) Fauna Ground Counts Module. In ‘Ecological Field Monitoring Protocols Manual using the Ecological Monitoring System Australia’. (Eds S O’Neill, K Irvine, A Tokmakoff, B Sparrow). TERN, Adelaide." ;
     rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
-    skos:definition "The compass direction (e.g., N, E, NW, SE, etc.) facing the intended outlook zone from a vantage point." ;
-    skos:prefLabel "outlook zone direction" ;
-    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/attribute_concepts/outlook-zone-direction.ttl"^^xsd:anyURI ;
-    tern:valueType tern:Text ;
+    skos:definition "The compass direction (left boundary, in degrees from north) facing the intended outlook zone from a vantage point." ;
+    skos:prefLabel "outlook zone - left boundary direction" ;
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/attribute_concepts/outlook-zone-left-direction.ttl"^^xsd:anyURI ;
+    tern:valueType tern:Degrees ;
```
Comment (junrongYu) [inline vocab_files/attribute_concepts/outlook-zone-left-direction.ttl:15]:
```suggestion
    tern:valueType tern:Float ;
```

File: vocab_files/attribute_concepts/outlook-zone-right-direction.ttl [hunk 2]
```diff
@@ -0,0 +1,17 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX tern: <https://w3id.org/tern/ontologies/tern/>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/a73f5249-542d-594f-b7d8-eb074c5405dc>
```
Comment (junrongYu) [inline vocab_files/attribute_concepts/outlook-zone-right-direction.ttl:8]:
```suggestion
<https://linked.data.gov.au/def/nrm/ea51c3ab-1d1e-48f7-ae83-8d06d4f97d04>
```
We need a new UUID here for a new concept

File: vocab_files/attribute_concepts/outlook-zone-right-direction.ttl [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    a skos:Concept ;
+    dcterms:source "McCallum K, Bignall J, Cox B, O’Neill S, Sparrow B. (2023) Fauna Ground Counts Module. In ‘Ecological Field Monitoring Protocols Manual using the Ecological Monitoring System Australia’. (Eds S O’Neill, K Irvine, A Tokmakoff, B Sparrow). TERN, Adelaide." ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:definition "The compass direction (right boundary, in degrees from north) facing the intended outlook zone from a vantage point." ;
+    skos:prefLabel "outlook zone - right boundary direction" ;
+    schema:url "https://github.com/ternaustralia/dawe-rlp-vocabs/tree/main/vocab_files/attribute_concepts/outlook-zone-right-direction.ttl"^^xsd:anyURI ;
+    tern:valueType tern:Degrees ;
```
Comment (junrongYu) [inline vocab_files/attribute_concepts/outlook-zone-right-direction.ttl:15]:
```suggestion
    tern:valueType tern:Float ;
```
Degree's value type is Float in our information model.

File: vocab_files/observable_property_concepts/rainfall-duration.ttl
Comment (junrongYu) [inline vocab_files/observable_property_concepts/rainfall-duration.ttl:1]: Hi @arunherb , can you please update all the metadata files names as well? For both `rainfall duration` and `rainfall type`

**Review by junrongYu — CHANGES_REQUESTED** [review]

**Comment by junrongYu** [comment]
Hi @arunherb , can you please double check the attributes collection file for this module. Cuz many attributes have been updated and new ones are added, not sure whether they belong to this module.

File: vocab_files/attribute_concepts/outlook-zone-right-direction.ttl (hunk 2 above)
Comment (arunherb) [reply vocab_files/attribute_concepts/outlook-zone-right-direction.ttl:8]: Done

File: vocab_files/attribute_concepts/outlook-zone-right-direction.ttl (hunk 3 above)
Comment (arunherb) [reply vocab_files/attribute_concepts/outlook-zone-right-direction.ttl:15]: Done

File: vocab_files/observable_property_concepts/rainfall-duration.ttl
Comment (arunherb) [reply vocab_files/observable_property_concepts/rainfall-duration.ttl:1]: Done

**Comment by arunherb** [comment]
> Hi @arunherb , can you please double check the attributes collection file for this module. Cuz many attributes have been updated and new ones are added, not sure whether they belong to this module.

Done

**Comment by junrongYu** [comment]
Hi @arunherb , thanks for the updates, can you please update the metadata file of OP `animal count` in sub protocol `vantage point`?, the collection URI, module URI, and file path etc. are from Transect protocol

**Comment by arunherb** [comment]
> Hi @arunherb , thanks for the updates, can you please update the metadata file of OP `animal count` in sub protocol `vantage point`?, the collection URI, module URI, and file path etc. are from Transect protocol

Done

**Review by junrongYu — CHANGES_REQUESTED** [review]
Hi @arunherb , for 2 metadata files of OP `animal count` in both `transect` and `vantage point`, can you please bring the row `urnp:observablePropertiesCollection` back, it's accidentally removed.

**Comment by arunherb** [comment]
> Hi @arunherb , for 2 metadata files of OP `animal count` in both `transect` and `vantage point`, can you please bring the row `urnp:observablePropertiesCollection` back, it's accidentally removed.

Done

**Pull request merged**